src/fmt/mod.rs

```diff
@@ -70,6 +91,38 @@ pub enum TimestampPrecision {
     Nanos,
 }
 
+/// Style for displaying key/value structured data
+///
+/// This determines how key/value pairs are displayed in the log output.
+/// The values can be suppressed, included in a trailer at the end of the line, or
+/// included as multiple lines after the message.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum KVStyle {
```

Comment:
Is this a zombie from resolving the merge conflicts?

Reply:
ah, yes it is